config/puma.rb

```diff
@@ -1,4 +0,0 @@
-threads 8,250
```

Comment:
This file wasn't loaded by `rails server`, so it seems kind of pointless. Unless people actually run `puma` directly.